<launch>

  <arg name="namespace" default="ant01"/>

  <node pkg="swarm_bridge" name="drone_bridge_node1" type="bridge_node_ants" output="screen">
  
    <remap from="/state_estimation" to="/ant01/state_estimation"/>
    <remap from="/others_odom" to="/msg_in_bridge1"/>

    <param name="broadcast_ip" value="127.0.0.255" type="string"/>
    <!-- <param name="broadcast_ip" value="192.168.228.255" type="string"/> -->
    <param name="drone_id" value="1"/>
    <param name="drone_num" value="5"/>
    <param name="odom_max_freq" value="70"/>
    <param name="NameSpace" type="string" value="$(arg namespace)" />

  </node>

</launch>
